examples/amm/src/contract.rs

```diff
@@ -329,25 +363,132 @@ impl Amm {
         Ok(())
     }
 
+    async fn try_to_execute_swap_bids(&mut self) -> Result<(), AmmError> {
+        let mut swap_bid_ids_to_remove = Vec::new();
+        let swap_bid_ids = self.swap_bids.indices().await?;
+
+        for swap_bid_id in swap_bid_ids {
```

Comment:
If we iterate over all of them every time, I can cause a lot of additional fees for all future users just by adding a few swap bids that won't get executed.

If we filled them partially, we could sort them by price, but then we have rounding issues again. Not sure how swap bids are usually implemented.

Reply:
Like we discussed on Slack, I'll remove swap bids for now until we can figure out how to implement it